src/Tooltip/Tooltip.stories.tsx

```diff
@@ -26,7 +25,7 @@ storiesOf('Tooltip', module).add(
         ],
         'top-start',
       )}
-      trigger="manual"
+      trigger="click"
```

Comment:
Tooltip arrow seems to have a different background color as compared to tooltip body

Reply:
Fixed